contracts/cheddar_5x5_tic_tac_toe/src/lib.rs

```diff
@@ -1040,55 +1048,55 @@ mod tests {
         assert!(ctr.get_active_games().contains(&(game_id, GameView::from(&game))));
 
         let mut tiles = make_move(&mut ctx, &mut ctr, &player_1, &game_id, 0, 0);
-        print_tiles(&tiles);
+        print_tiles(tiles);
         tiles = make_move(&mut ctx, &mut ctr, &player_2, &game_id, 0, 1);
-        print_tiles(&tiles);
+        print_tiles(tiles);
         tiles = make_move(&mut ctx, &mut ctr, &player_1, &game_id, 0, 2);
-        print_tiles(&tiles);
+        print_tiles(tiles);
         tiles = make_move(&mut ctx, &mut ctr, &player_2, &game_id, 0, 3);
-        print_tiles(&tiles);
+        print_tiles(tiles);
         tiles = make_move(&mut ctx, &mut ctr, &player_1, &game_id, 0, 4);
-        print_tiles(&tiles);
+        print_tiles(tiles);
```

Comment:
can we remove some of the print statements? I think we don't need them that many

Reply:
sure, lets just do one/two prints